src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -1,24 +1,26 @@
 /* eslint-disable no-unexpected-multiline */
 import * as React from 'react';
+import ContextMenu from 'react-native-context-menu-view';
 import {Dimensions, StyleSheet, Text, View} from 'react-native';
 
+import ChatContextMenuView from '../../ContextMenuView/ChatContextMenuView';
+import ChatItemAttachment from './ChatItemAttachment';
 import IconChatCheckMark from '../../../assets/icon/IconChatCheckMark';
 import IconChatClockGrey from '../../../assets/icon/IconChatClockGrey';
-import {ChatItemMyTextProps} from '../../../../types/component/AnonymousChat/BaseChatItem.types';
-import {ChatStatus} from '../../../../types/database/schema/ChannelList.types';
-import {SIGNED} from '../../../hooks/core/constant';
-import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
+import dimen from '../../../utils/dimen';
 import {
   AVATAR_MARGIN,
   BUBBLE_LEFT_PADDING,
-  BUBBLE_RIGHT_PADDING,
   BUBBLE_LEFT_PADDING_ATTACHMENT,
+  BUBBLE_RIGHT_PADDING,
   BUBBLE_RIGHT_PADDING_ATTACHMENT
 } from './ChatItemAttachmentStyles';
-import ChatItemAttachment from './ChatItemAttachment';
-import dimen from '../../../utils/dimen';
+import {COLORS} from '../../../utils/theme';
+import {ChatItemMyTextProps} from '../../../../types/component/AnonymousChat/BaseChatItem.types';
+import {ChatStatus} from '../../../../types/database/schema/ChannelList.types';
 import {LinkableText} from '../../LinkableText';
+import {SIGNED} from '../../../hooks/core/constant';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ContextMenu` from 'react-native-context-menu-view' is not used in the code. It's a good practice to remove unused imports as they can lead to confusion and unnecessary increase in bundle size.
```diff
- import ContextMenu from 'react-native-context-menu-view';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->